src/app/manager-dashboard/manager.service.ts

```diff
@@ -77,7 +77,7 @@ export class ManagerService {
     return this.couchService.post('admin_activities', { ...log, type });
   }
 
-  getLogs() {
+  getLogs(tillDate?: number) {
```

Comment:
You've added this parameter to the function but it's never used.  I think this is not complete.